dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/context/DubboConfigBeanInitializer.java

```diff
@@ -32,56 +33,26 @@
 import org.springframework.beans.FatalBeanException;
 import org.springframework.beans.factory.BeanFactory;
 import org.springframework.beans.factory.BeanFactoryAware;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.beans.factory.config.BeanPostProcessor;
+import org.springframework.beans.factory.InitializingBean;
 import org.springframework.beans.factory.config.ConfigurableListableBeanFactory;
-import org.springframework.context.ApplicationContext;
-import org.springframework.context.ApplicationContextAware;
-import org.springframework.core.Ordered;
 
 import java.util.concurrent.atomic.AtomicBoolean;
 
-import static org.springframework.beans.factory.BeanFactoryUtils.beansOfTypeIncludingAncestors;
 
 /**
  *
- * Post-processor Dubbo config bean initialization.
+ * Dubbo config bean initializer.
  *
  * NOTE: Dubbo config beans MUST be initialized after registering all BeanPostProcessors,
  * that is after the AbstractApplicationContext#registerBeanPostProcessors() method.
  */
-public class DubboConfigInitializationPostProcessor implements BeanPostProcessor, BeanFactoryAware, ApplicationContextAware, Ordered {
+public class DubboConfigBeanInitializer implements BeanFactoryAware, InitializingBean {
 
-    public static String BEAN_NAME = "dubboConfigInitializationPostProcessor";
-
-    /**
-     * This bean post processor should run before seata GlobalTransactionScanner(1024)
-     */
-    @Value("${dubbo.config-initialization-post-processor.order:1000}")
-    private int order = 1000;
+    public static String BEAN_NAME = "dubboConfigInitializer";
```

Comment:
Should it be change to `dubboConfigBeanInitializer`?

Reply:
Fixed it